src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -50,6 +81,32 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 AssemblyVersionMajorTextBox, AssemblyVersionMinorTextBox, AssemblyVersionBuildTextBox, AssemblyVersionRevisionTextBox}
         End Sub
 
+        ''' <summary>
+        ''' This checks the properties for licenses to determine the initial state of the licensing section.
+        ''' Currently, If both are set, it will default to just enabling the PackageLicenseExpression.
```

Comment:
When referencing symbols in doc comments, link to types (eg. `<see cref="PackageLicenseExpression" />`) so that refactorings keep things up to date.

Reply:
Good point, fixed.